conf/before_components.mk

```diff
@@ -8,42 +8,52 @@
 # FV3, CCPP, FMS, MOM6, CICE, WW3 or NEMS.  Otherwise, applications will
 # break.  There should only be logic specific this coupled app. 
 #
-# Logic specific to FV3, CCPP, FMS, or NEMS belong in NEMS/src/incmake.
+# Logic specific to FV3, CCPP, FMS, WW3, or NEMS belong in NEMS/src/incmake.
 
 # ----------------------------------------------------------------------
 # Decide the conf and modulefile names.
 # S2S_DEBUG_MODULE is defined in GNUmakefile
-ifeq ($(S2S_DEBUG_MODULE),false)
-  CHOSEN_MODULE=$(BUILD_TARGET)/fv3_coupled
-else
+
+#CHOSEN_MODULE=$(BUILD_TARGET)/fv3
+CHOSEN_MODULE=$(BUILD_TARGET)/fv3_coupled
+
+ifeq ($(S2S_DEBUG_MODULE),true)
   CHOSEN_MODULE=$(BUILD_TARGET)/fv3_coupled_debug
 endif
 
 #$(info CHOSEN_MODULE is $(CHOSEN_MODULE))
 
-CONFIGURE_NEMS_FILE=configure.fv3_coupled.$(BUILD_TARGET)
+ifneq (,$(findstring INTEL16=Y,$(FV3_MAKEOPT)))
```

Comment:
I think we eliminated intel 16. So we may not need this statement, also we don't suggest CCPP to run with intel 16.

Reply:
I will remove this if block.

Reply:
If block removed (f542289).